packages/ui/src/Modal.tsx

```diff
@@ -128,7 +128,7 @@ const ModalContent: FC<{
         </View>
       )}
       {children && (
-        <View aria-role="text" style={{marginTop: text ? 0 : 12}}>
+        <View aria-role="text" style={{marginTop: text ? 0 : 12, width: "100%"}}>
```

Comment:
For some reason, now if you click anywhere on the modal it closes it and is unclear if it just closed it or considered it 'confirmed'.

Reply:
Great catch. It's actually because the modal is nested within the buttons. Any click on the overlay from the modal is bubbling up to the button. The larger issue here is that we were treating a click outside of the modal as a confirmation. I'm just removing the logic to dismiss the modal with a click outside of the modal for now. I think in the future, we'd want a modalService that we can call as opposed to nesting a modal and it's overlay into any component we need a modal with. Not today though lol